### PR TITLE
Changed the starter FW mission

### PR DESCRIPTION
**Pull request by Gkramme**
Made the starting Katya mission a bit less confusing.

### CODE REVIEW

**Comment by frymaster** [comment]
Noob here, I got confused in issue #741 :)

**Comment by Pointedstick** [comment]
s/anymore/any more/

File: data/free worlds intro.txt [hunk 1]
```diff
@@ -485,9 +485,17 @@ mission "FW Katya 1"
...
-					decline
-	
+					goto `sorry`
+
+                        label `sorry`
+	                `      "I'm sorry about that, and I assure you it will not happen again. As I said earlier, I don't want to hire people who will lash out when their cargo hold is scanned."`
+                        choice
+                                `       "I understand. What do you need me to do?`
+                                        goto `more`
+                                `       "Still, I am not intrested in assisting the Free Worlds. Please do not offer me anymore work in the future."`
```
Comment (Pointedstick) [inline data/free worlds intro.txt]: s/anymore/any more/

File: data/free worlds intro.txt [hunk 2]
```diff
@@ -485,9 +485,17 @@ mission "FW Katya 1"
...
-					decline
-	
+					goto `sorry`
+
+                        label `sorry`
+	                `      "I'm sorry about that, and I assure you it will not happen again. As I said earlier, I don't want to hire people who will lash out when their cargo hold is scanned."`
+                        choice
+                                `       "I understand. What do you need me to do?`
+                                        goto `more`
+                                `       "Still, I am not intrested in assisting the Free Worlds. Please do not offer me any more work in the future."`
```
Comment (mrec) [inline data/free worlds intro.txt]: Typo: "intrested"

**Comment by Gkramme** [comment]
Fixed already.

**Comment by Bladewood** [comment]
Oops, did not notice the other comments.

**Comment by endless-sky** [comment]
I like the idea of this change, but it needs a few fixes. There's a missing close quotation mark after "What do you need me to do?", and also the response to that ("Okay, here's the deal...") doesn't flow very well.

Also, this PR seems to also include some outfit changes, one of which isn't correct: "it is" was already correct, and "its" is not - it would be "it's" if you're switching to a contraction there, but there's no real reason to.

**Comment by Gkramme** [comment]
@endless-sky, sorry about that. I was going to make a seprate PR, but it commited to this

**Comment by Gkramme** [comment]
Closing this because it was bad

**Pull request closed without merging**